src/Service/EndpointService.php

```diff
@@ -516,7 +517,7 @@ private function getNormalPath(array $parameters): array
 
         try {
             $combinedArray = array_combine($path, explode('/', $pathRaw));
-        } catch (Exception $exception) {
+        } catch (ValueError $exception) {
```

Comment:
Why only check for ValueErrors here?
Shouldn't we catch all type of exceptions array_combine() and explode() can throw?

Reply:
ValueError seems to be the only exception that reasonably can be thrown here. Catching exception will _not_ catch ValueError, hence the change.